### PR TITLE
docs: Update apps plugin docs for TAP 1.5

### PR DESCRIPTION
**Pull request by warango4**
# Which other branches do you want a technical writer to cherry-pick this PR to (if any)?

TAP 1.5

### CODE REVIEW

File: cli-plugins/apps/command-reference/workload_create_update_apply.hbs.md [hunk 1]
```diff
@@ -106,7 +106,7 @@ This is the application the workload is part of. This is part of the workload me
 <details><summary>Example</summary>
 
 ```console
-tanzu apps workload apply tanzu-app --git-repo https://github.com/vmware-tanzu/application-accelerator-samples --sub-path tanzu-java-web-app --git-tag tap-1.3 --type web --app tanzu-java-web-app
+tanzu apps workload apply tanzu-app --git-repo https://github.com/vmware-tanzu/application-accelerator-samples --sub-path tanzu-java-web-app --git-tag tap-1.5.0 --type web --app tanzu-java-web-app
```
Comment (shaheerkootteeri) [inline cli-plugins/apps/command-reference/workload_create_update_apply.hbs.md:109]: are there any extra space here ?

File: cli-plugins/apps/command-reference/workload_create_update_apply.hbs.md [hunk 2]
```diff
@@ -396,17 +403,19 @@ tanzu apps workload apply tanzu-java-web-app --git-repo https://github.com/vmwar
...
-Commit in Git repository from where the workload is resolved. Can be used with `--git-branch` or `git-tag`.
+Commit in Git repository from where the workload is resolved. Either `--git-branch` or `--git-tag`
+can be specified with it too.
+It can be unset by defining it as empty string when applying a workload (`--git-commit ""`).
 
 <details><summary>Example</summary>
 
 ```console
-tanzu apps workload apply tanzu-java-web-app --git-repo https://github.com/vmware-tanzu/application-accelerator-samples --sub-path tanzu-java-web-app --git-tag tap-1.3 --git-commit 1c4cf82e499f7e46da182922d4097908d4817320 --type web
+tanzu apps workload apply tanzu-java-web-app --git-repo https://github.com/vmware-tanzu/application-accelerator-samples --sub-path tanzu-java-web-app --git-commit 1c4cf82e499f7e46da182922d4097908d4817320 --type web
```
Comment (shaheerkootteeri) [inline cli-plugins/apps/command-reference/workload_create_update_apply.hbs.md:418]: tap version is missing in the command

File: cli-plugins/apps/create-workload.hbs.md [hunk 3]
```diff
@@ -48,6 +48,15 @@ Where:
 - `--git-branch` (optional) specifies which branch in the repository to pull the code from.
 - `--type` distinguishes the workload type.
 
+This process can also be done with non-publicly accessible repositories. These require authentication
+using credentials stored in a Kubernetes secret. The supply chain is in charge of manage these credentials.
```
Comment (shaheerkootteeri) [inline cli-plugins/apps/create-workload.hbs.md:52]: nit : `managing` these credentials ?

File: cli-plugins/apps/command-reference/workload_create_update_apply.hbs.md (hunk 1 above)
Comment (warango4) [reply cli-plugins/apps/command-reference/workload_create_update_apply.hbs.md:109]: No, the spaces are correct.

File: cli-plugins/apps/command-reference/workload_create_update_apply.hbs.md (hunk 2 above)
Comment (warango4) [reply cli-plugins/apps/command-reference/workload_create_update_apply.hbs.md:418]: No need for tag, already removed

File: cli-plugins/apps/create-workload.hbs.md (hunk 3 above)
Comment (warango4) [reply cli-plugins/apps/create-workload.hbs.md]: updated to use the word `managing`

**Review by odinnordico — APPROVED** [review]

**Pull request merged**